catalogue/webapp/components/ImageViewer/ImageViewer.js

```diff
@@ -126,27 +119,13 @@ type ImageViewerState = {|
 |}
 
 class ImageViewer extends React.Component<ImageViewerProps, ImageViewerState> {
-  handleViewerDisplay: Function;
-  viewButtonMountedHandler: Function;
-
-  constructor(props: ImageViewerProps) {
-    super(props);
-    this.state = {
-      showViewer: false,
-      mountViewButton: false,
-      viewButtonMounted: false
-    };
-    this.handleViewerDisplay = this.handleViewerDisplay.bind(this);
-    this.viewButtonMountedHandler = this.viewButtonMountedHandler.bind(this);
-  }
-
-  viewButtonMountedHandler() {
-    this.setState(prevState => ({
-      viewButtonMounted: !prevState.viewButtonMounted
-    }));
-  }
+  state = {
+    showViewer: false,
+    mountViewButton: false,
```

Comment:
This as a boolean state name doesn't make much sense to me?
Could we keep to `isViewButtonMounted` and (I think this should be) `shouldMountViewButton`?

Reply:
(or if you prefer not to as this component needs a bit of love/)